<ui version="4.0" >
 <class>VP1IncomingMessageForm</class>
 <widget class="QWidget" name="VP1IncomingMessageForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>6</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>22</number>
          </property>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="1" column="0" >
             <widget class="QPushButton" name="pushButton_allclear" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>1</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>16777215</width>
                <height>18</height>
               </size>
              </property>
              <property name="text" >
               <string>Clear</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2" >
             <widget class="QPushButton" name="pushButton_allblock" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>1</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>16777215</width>
                <height>18</height>
               </size>
              </property>
              <property name="text" >
               <string>Block</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2" >
             <widget class="QLCDNumber" name="lcdNumber_pending" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>0</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape" >
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow" >
               <enum>QFrame::Raised</enum>
              </property>
              <property name="smallDecimalPoint" >
               <bool>true</bool>
              </property>
              <property name="numDigits" >
               <number>3</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2" >
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Pending messages:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="1" column="1" colspan="2" >
             <widget class="QPushButton" name="pushButton_senderblock" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>1</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>16777215</width>
                <height>18</height>
               </size>
              </property>
              <property name="text" >
               <string>Block</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QPushButton" name="pushButton_senderclear" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>1</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>16777215</width>
                <height>18</height>
               </size>
              </property>
              <property name="text" >
               <string>Clear</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2" >
             <widget class="QLCDNumber" name="lcdNumber_pendingsender" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>0</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape" >
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow" >
               <enum>QFrame::Raised</enum>
              </property>
              <property name="smallDecimalPoint" >
               <bool>true</bool>
              </property>
              <property name="numDigits" >
               <number>3</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2" >
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>From this sender:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="2" >
             <widget class="QLCDNumber" name="lcdNumber_pendingexactsame" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>0</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape" >
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow" >
               <enum>QFrame::Raised</enum>
              </property>
              <property name="smallDecimalPoint" >
               <bool>true</bool>
              </property>
              <property name="numDigits" >
               <number>3</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2" >
             <widget class="QPushButton" name="pushButton_messageblock" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>1</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>16777215</width>
                <height>18</height>
               </size>
              </property>
              <property name="text" >
               <string>Block</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2" >
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>Same exact message:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QPushButton" name="pushButton_messageclear" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>1</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>16777215</width>
                <height>18</height>
               </size>
              </property>
              <property name="text" >
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_header" >
       <property name="text" >
        <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Incoming message from &lt;span style=" font-weight:600;">SENDERPLACEHOLDER&lt;/span>:&lt;/p>&lt;/body>&lt;/html></string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser_message" >
     <property name="minimumSize" >
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="textInteractionFlags" >
      <enum>Qt::TextSelectableByMouse</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" native="1" name="widget_showtab" >
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_showtab" >
          <property name="text" >
           <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">It is requested that you look at the tab named &lt;span style=" font-weight:600;">DUMMYTABNAME&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_showtab_gothere" >
          <property name="text" >
           <string>&amp;Go there now</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_showtaberror" >
          <property name="text" >
           <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:12pt; color:#ff0000;">Unfortunately such a tab was not found!&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_dismiss" >
       <property name="text" >
        <string>&amp;Dismiss</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton_showtab_gothere</tabstop>
  <tabstop>pushButton_dismiss</tabstop>
  <tabstop>pushButton_allclear</tabstop>
  <tabstop>pushButton_allblock</tabstop>
  <tabstop>pushButton_senderclear</tabstop>
  <tabstop>pushButton_senderblock</tabstop>
  <tabstop>pushButton_messageclear</tabstop>
  <tabstop>pushButton_messageblock</tabstop>
  <tabstop>textBrowser_message</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_dismiss</sender>
   <signal>clicked()</signal>
   <receiver>VP1IncomingMessageForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>571</x>
     <y>624</y>
    </hint>
    <hint type="destinationlabel" >
     <x>444</x>
     <y>630</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
